src/modules/avformat/common.h

```diff
@@ -22,6 +22,9 @@
 
 #include <framework/mlt.h>
 #include <libswscale/swscale.h>
+#include <libavutil/frame.h>
+
+#define MLT_SWS_FLAGS "bicubic+accurate_rnd+full_chroma_int+full_chroma_inp"
```

Comment:
`MLT_SWS_FLAGS` is very similar to `mlt_get_sws_flags`. Since it for use with avfilter-related things consider to change its name to `MLT_AVFILTER_SWS_FLAGS`

Reply:
Good idea. Done